bin/prep-release.php

```diff
@@ -190,6 +190,7 @@ function usage() {
     "%RELEASE_VERSION%" => $VERSION,
     "%RELEASE_STABILITY%" => $STABILITY,
     "%RELEASE_FILES%" => join("\n", $TREE),
+    "%RELEASE_NOTES%" => file_exists(__DIR__ . '/../changelog') ? file_get_contents(__DIR__ . '/../changelog') : '',
```

Comment:
Consider making this an optional script argument instead of hard-coding a file. That would remove coupling between this script and the GitHub Actions.

Reply:
I used a file in order to avoid dealing with escaping a multi-line string in the shell, especially as it contains potentially user-submitted data (e.g. pull request titles). The generated notes are only ever passed through files to ensure we don't open ourselves up to any script injection or data leaks due to shell expansions (e.g. because somebody placed `$GITHUB_ENV` in a pull request title which could then be expanded into the entire env value.

Reply:
Good point. I've added an optional parameter to the script and updated the makefile and package-release workflow.